docs/install.rst

```diff
@@ -200,23 +200,36 @@ Operating System Packages
 
 **Debian**
 
-If you run Debian Stretch or Debian Sid, you can install certbot packages.
+If you run Debian Buster or Debian testing/sid, you can easily install certbot
+packages through commands like:
 
 .. code-block:: shell
 
    sudo apt-get update
-   sudo apt-get install certbot python-certbot-apache
+   sudo apt-get install certbot
+
+If you run Debian Stretch, we recommend you use the packages in Debian
+backports repository. First you'll have to follow the instructions at
+http://backports.debian.org/Instructions/ to enable the Stretch backports repo,
+if you have not already done so. Then run:
+
+.. code-block:: shell
 
-If you don't want to use the Apache plugin, you can omit the
-``python-certbot-apache`` package. Or you can install ``python-certbot-nginx`` instead.
+   sudo apt-get install certbot -t stretch-backports
 
-Packages exist for Debian Jessie via backports. First you'll have to follow the
-instructions at http://backports.debian.org/Instructions/ to enable the Jessie backports
-repo, if you have not already done so. Then run:
+In all of these cases, there also packages available to help Certbot integrate
```

Comment:
Maybe we should emphasize this more in order to get users to pay attention to the fact that they will probably need at least one of these additional packages. (At a minimum, we could say "Almost all users will want to install packages that help Certbot integrate with other software" or something like that.)